include/gridtools/fn/common_interface.hpp

```diff
@@ -30,4 +33,9 @@ namespace gridtools::fn {
         return {std::forward<Args>(args)...};
     }
 
+    template <class D>
+    constexpr auto index(D) {
+        return gridtools::stencil::positional<std::decay_t<D>>();
```

Comment:
```suggestion
        return gridtools::stencil::positional<D>();
```
Or is there any case where we could need the `std::decay_t`?

Reply:
yes...